examples/cli/cli_demo.go

```diff
@@ -550,6 +567,35 @@ func readCommand(cmd string) {
 	}
 }
 
+func setToken(args []string) {
+	pn.SetToken(args[0])
+}
+
+//p0F2AkF0Gl2AX-JDdHRsCkNyZXOkRGNoYW6gQ2dycKBDdXNyoWl1LTMzNTIwNTUPQ3NwY6Fpcy0xNzA3OTgzGB9DcGF0pERjaGFuoENncnCgQ3VzcqBDc3BjoERtZXRhoENzaWdYINqGs2EyEMHPZrp6znVqTBzXNBAD_31hUH3JuUSWE2A6
```

Comment:
Should these be removed?

Reply:
A token as a example in SetToken method. TTL is already expired.

Reply:
Removed